tests/acceptance/user-query-test.js

```diff
@@ -0,0 +1,111 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'plenario-explorer/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | user query');
+
+test('Front page loads properly with query parameters.', function(assert) {
+  visit('?obs_date__ge=2010-06-01&obs_date__le=2017-07-02&agg=day&center=seattle');
+
+  andThen(function(){
+    assert.equal(currentRouteName(), 'discover.index', "The front page routes to discover.index.");
+    assert.equal($('#submit-query').is('button'), true, "The front page renders properly: submit button is present.");
+    assert.equal($('#start-date-filter input').val(), '06/01/2010', "Query parameter was able to set the start date.");
+    assert.equal($('#end-date-filter input').val(), '07/02/2017', "Query parameter was able to set the end date.");
+    assert.equal($('#agg-select option:selected').text().trim(), 'day', "Query parameter was able to set the aggregation mode.");
+    assert.equal($('#map-center-select option:selected').text().trim(), 'Seattle', "Query parameter was able to set the map center.");
+  });
+});
+
+test('User can make a query.', function(assert) {
+  visit('discover?location_geom__within=%7B"type"%3A"Feature"%2C"properties"%3A%7B%7D%2C"geometry"%3A%7B"type"%3A"Polygon"%2C"coordinates"%3A%5B%5B%5B-87.68862247467041%2C41.80510182643331%5D%2C%5B-87.68862247467041%2C41.90432124806034%5D%2C%5B-87.55678653717041%2C41.90432124806034%5D%2C%5B-87.55678653717041%2C41.80510182643331%5D%2C%5B-87.68862247467041%2C41.80510182643331%5D%5D%5D%7D%7D');
```

Comment:
I just needed an arbitrary geometry so that `/discover/aggregate` would accept the query. Should I make it more readable, or should I just note that it's arbitrary?